operator/pkg/util/progress/progress.go

```diff
@@ -201,6 +204,8 @@ func (p *ManifestLog) ReportFinished() {
 	if p == nil {
 		return
 	}
+	p.mu.Lock()
+	defer p.mu.Unlock()
```

Comment:
Seems this lock should not protect report, because the `report` function is already protected by this lock, not sure how the test passed, i am not very familiar with this code

Reply:
i agree, let me fix it